src/utils/args.ts

```diff
@@ -75,9 +76,7 @@ class ArgsAccount extends Args {
 
     isValid = () => ArgsAccount.isValid(this);
 
-    toNet = () => this.value.split(".").pop() === "testnet" ? "testnet" : "mainnet";
-
-    toUrl = (net = this.toNet()) => `https://explorer.${net}.near.org/accounts/${this.value}`;
+    toUrl = (network: string) => `https://explorer.${network}.near.org/accounts/${this.value}`;
```

Comment:
Why remove the `toNet()` function?

Reply:
Not all addresses have .near or .testnet at the end. Example: [aurora](https://explorer.near.org/accounts/aurora) or dev addresses like [this one](https://explorer.near.org/accounts/b506a8ed12bd3c9910d71c519bed05958e623d19c256e1373fae80fae997ad56)

Reply:
It supports this behavior.

Reply:
not sure. I think your function will always return aurora or public key addresses as mainnet addresses. Which is not the case

Reply:
The function cannot return anything but `"testnet"` or `"mainnet"`. Please tell me if I'm wrong, but if the network does not end with `.testnet` we know it is a mainnet address, is that not true?

Reply:
`aurora` and pubKey addresses are valid for mainnet AND testnet.
example: https://explorer.testnet.near.org/accounts/dev-1653348739484-27226161167116